### ISSUE TITLE
Object Folder cannot be restored from recycle bin

### ISSUE THREAD

**Issue by beheist** [pimcore/pimcore#6069]
# Bug Report
Object Folder cannot be restored from recycle bin after deleting it.
The following error is displayed:
```
Timestamp: Fri Mar 20 2020 09:29:01 GMT+0100 (Mitteleuropäische Normalzeit)
Status: 500 | Internal Server Error
URL: /admin/recyclebin/restore
Method: POST
Message: Warning: Invalid argument supplied for foreach()
Trace: 
#0 C:\src\yaskawa\pimcore\vendor\pimcore\pimcore\models\Element\Recyclebin\Item.php(158): Pimcore\Model\Element\Recyclebin\Item->restoreChilds(Object(Pimcore\Model\DataObject\Folder))
#1 C:\src\yaskawa\pimcore\vendor\pimcore\pimcore\bundles\AdminBundle\Controller\Admin\RecyclebinController.php(142): Pimcore\Model\Element\Recyclebin\Item->restore()
#2 C:\src\yaskawa\pimcore\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(151): Pimcore\Bundle\AdminBundle\Controller\Admin\RecyclebinController->restoreAction(Object(Symfony\Component\HttpFoundation\Request))
#3 C:\src\yaskawa\pimcore\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 C:\src\yaskawa\pimcore\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php(198): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 C:\src\yaskawa\pimcore\web\app.php(36): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 {main}
```

### Expected behavior
Folder should be restored to the object structure.

### Actual behavior
Error message.

### Steps to reproduce
1. Create an object folder with items in it
2. Delete te folder and try to restore it

The issue was tested in Pimcore 6.2.3 on Windows and CentOS.

**Comment by weisswurstkanone** [comment]
Already fixed in a later version.

**Comment by brusch** [comment]
Fixed by #5237 in v6.3.0
